package.json

```diff
@@ -56,15 +57,15 @@
     "not op_mini all"
   ],
   "devDependencies": {
-    "@babel/core": "^7.13.15",
+    "@babel/core": "^7.17.8",
+    "@babel/eslint-parser": "^7.17.0",
```

Comment:
I thought that using the `concurrent_offers` value from `nijobs-commons` instead of a hard coded number would be better. Changing the `rules.json` file to a JS file and importing the constants was failing to compile, this solved that issue. It was @imnotteixeira that did it so he might know the reasoning behind this better than me (although I read somewhere that the `es-lint` babel plugin that we had been using is old/deprecated, but don't quote me on that)